build.gradle

```diff
@@ -33,7 +33,7 @@ plugins {
 
 apply plugin: 'base'
 
-ext.attachVersion = "4.0.5"
+ext.attachVersion = "4.0.11"
```

Comment:
This is part of #56 as well, that should get integrated before this one.

Reply:
Sure, go ahead?

Reply:
maybe integrate #58 first?

Reply:
done